examples/thompson_sampling.py

```diff
@@ -206,32 +213,19 @@ def choose_next_sample(self, n_step=2_000):
         self.X = np.append(self.X, next_sample)
         self.y = np.append(self.y, next_observation)
 
-        # return everything if possible
```

Comment:
what is your logic here? why not return nothing if it errors and revert to picking a random point in the domain?

Reply:
This is in there so that the plots still make sense even when we plot the samples during the random exploration phase.